{"identifier":{"url":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent"]]},"kind":"symbol","sections":[],"schemaVersion":{"minor":3,"patch":0,"major":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)"]}],"abstract":[{"type":"text","text":"Invoked when a rest call fails for authentication reasons."}],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"authenticationFailed"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"kind":"text","text":")"}]}]},{"kind":"parameters","parameters":[{"content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"The error that caused the authentication failure."}]}],"name":"error"}]}],"metadata":{"title":"ZendeskEvent.authenticationFailed(error:)","symbolKind":"case","externalID":"s:10ZendeskSDK0A5EventO20authenticationFailedyACs5Error_p_tcACmF","roleHeading":"Case","role":"symbol","modules":[{"name":"ZendeskSDK"}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"authenticationFailed"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:s5ErrorP","text":"Error","kind":"typeIdentifier"},{"text":")","kind":"text"}]},"references":{"doc://com.zendesk.Zendesk/documentation/ZendeskSDK":{"identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","title":"ZendeskSDK","type":"topic","role":"collection","abstract":[],"url":"\/documentation\/zendesksdk","kind":"symbol"},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent":{"navigatorTitle":[{"text":"ZendeskEvent","kind":"identifier"}],"type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ZendeskEvent"}],"abstract":[{"text":"The set of events that can be emitted from Zendesk SDK.","type":"text"}],"title":"ZendeskEvent","identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent","role":"symbol","kind":"symbol","url":"\/documentation\/zendesksdk\/zendeskevent"},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent/authenticationFailed(error:)":{"title":"ZendeskEvent.authenticationFailed(error:)","url":"\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)","role":"symbol","kind":"symbol","type":"topic","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"error"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"abstract":[{"type":"text","text":"Invoked when a rest call fails for authentication reasons."}],"identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)"}}}